lib/accept.mli

```diff
@@ -43,13 +56,14 @@ type encoding =
   | Identity
   | AnyEncoding with sexp
 
+(** Language tags, as a list of subtags.
+    ["en-gb"] is represented as [Language ["en"; "gb"]].
+    @see <http://www.w3.org/Protocols/rfc2616/rfc2616-sec14.html#sec14.4> The spec.
+*)
 type language = Accept_types.language =
```

Comment:
Note that this is formally a BCP47 *basic language range* tag and not a *language tag* (FWIW I do have a module [here](https://github.com/dbuenzli/remat/blob/master/src-common/nlang.mli) to handle these that I expect to release as an independent module at some point).  Also referring to https://tools.ietf.org/html/rfc7231#section-5.3.5 would be nicer, the latter obsoletes 2616 and makes a lot of things more clear notably in this case it links to the right RFC.